lib/asm/exception/__osException.s

```diff
@@ -191,8 +191,8 @@ glabel __osException
 /* 0D3B2C 800D2F2C 3C01800E */  lui   $at, %hi(__osHwIntTable) # $at, 0x800e
 /* 0D3B30 800D2F30 002A0821 */  addu  $at, $at, $t2
 /* 0D3B34 800D2F34 8C2A48A0 */  lw    $t2, %lo(__osHwIntTable)($at)
-/* 0D3B38 800D2F38 3C1D8013 */  lui   $sp, %hi(leoDiskStack) # $sp, 0x8013
-/* 0D3B3C 800D2F3C 27BDAAE0 */  addiu $sp, %lo(leoDiskStack) # addiu $sp, $sp, -0x5520
+/* 0D3B38 800D2F38 3C1D8013 */  lui   $sp, %hi(leoDiscStack) # $sp, 0x8013
```

Comment:
This might be an exception. In computing terms  I believe we want disk here instead.

Reply:
Fixed